README.md

```diff
@@ -45,62 +34,78 @@ This is the implementation of the [IPFS repo spec](https://github.com/ipfs/specs
+                          ├─────────────────┤     ├─────────────────┤
+                          │    Datastore    │     │ LevelDatastore  │
+                          └─────────────────┘     └─────────────────┘
+
+
+
+┌────────────────────────────────────────┐          ┌────────────────────────────────────────┐
+│       IPFSRepo - Default Node.js       │          │       IPFSRepo - Default Browser       │
+└────────────────────────────────────────┘          └────────────────────────────────────────┘
+
+
+
+            ┌─────────────────┐                                 ┌─────────────────┐
+            │                 │                                 │                 │
+            │        /        │                                 │        /        │
+            │                 │                                 │                 │
+            ├─────────────────┤                                 ├─────────────────┤
+            │   FsDatastore   │                                 │LevelJSDatastore │
+            └─────────────────┘                                 └─────────────────┘
+                     │                                                   │
+         ┌───────────┴───────────┐                           ┌───────────┴───────────┐
+         │                       │                           │                       │
+         │                       │                           │                       │
+┌─────────────────┐     ┌─────────────────┐         ┌─────────────────┐     ┌─────────────────┐
+│                 │     │                 │         │                 │     │                 │
+│     /blocks     │     │   /datastore    │         │     /blocks     │     │   /datastore    │
+│                 │     │                 │         │                 │     │                 │
+├─────────────────┤     ├─────────────────┤         ├─────────────────┤     ├─────────────────┤
+│ FlatfsDatastore │     │LevelDBDatastore │         │LevelJSDatastore │     │LevelJSDatastore │
+└─────────────────┘     └─────────────────┘         └─────────────────┘     └─────────────────┘
 ```
 
-This provides a well defined interface for creating and interacting with an IPFS
-Repo backed by a group of abstract backends for keys, configuration, logs, and
-more. Each of the individual repos has an interface defined by
-[interface-pull-blob-store](https://github.com/ipfs/interface-pull-blob-store): this
-enables us to make IPFS Repo portable (running on Node.js vs the browser) and
-accept different types of storage mechanisms for each repo (fs, levelDB, etc).
-
-### Good to know (historical context)
-
-- The datastore folder holds the legacy version of datastore, still built in levelDB, there is a current endeavour of pushing it to fs completely.
-- The blocks folder is the current version of datastore.
-- The keys repo doesn't exist yet, as the private key is simply stored inside config
```

Comment:
the first item is also not true, datastore is not legacy, it's being actively used and things added to it